dist-git-to-src-git/updates.md

```diff
@@ -0,0 +1,126 @@
+# Updates of source-git repos
+
+
+## Problem
+
+- We need to convert new dist-git commits to the existing source-git repository.
+- The changes need to be reversible.
+  - => We need to get back to the dist-git structure from the source-git.
+- Possible changes:
+  - changes in the packaging files (not in spec-file, not in patch files; e.g. config-files)
```

Comment:
minor point: every change usually results in changing a spec file as well (bump release number, new line in changelog)

Reply:
Good point.